src/plugins/custom_import_map/.eslintrc

```diff
@@ -0,0 +1,7 @@
+---
+extends:
```

Comment:
I am curious why we put .eslintrc inside of src/plugins/custom_import_map?
There are two ways for plugin to run eslint:
1) re-use OpenSearch-Dashboards eslint. In this situation, we don't need .eslintrc in the plugin. when run `yarn lint`, osd will run eslint for all the whole repo including installed plugins.
2) define a new .eslintrc which normally should just be under ./ and define some diff lint rules.
None of them should be in src folder.

Reply:
We want to have a structure where each plugin inside src/plugins/* has its own files and are importing the plugin `custom_import_map` within OSD which means all the files inside this folder will be taken under plugins folder of OSD. For these configs to work correctly, does it make sense to keep these files here?

Reply:
To me, I think it does not make a lot of sense to keep these config files inside src/plugins folder. These config should be applied to the repo. There might be two use cases 1) any contributors make some changes outside src 2) in the future we develop some code outside src.

Reply:
We will be adding each front-end plugin part of OSD maps within src/plugins folder in this repo which means every plugin will have its own config files. I'll definitely update the config files to inherit from the parent config files in OSD wherever required (in upcoming PRs), but for this PR I'll keep it as is and focus on merging the working functionality into the repo.